sphinx/source/docs/user_guide/bokehjs.rst

```diff
@@ -3,72 +3,70 @@
 Developing with JavaScript
 ==========================
 
-In order to create interactive plots and applications in the browser, Bokeh
-has a client-side library, BokehJS, to do all the work of drawing and rendering
-and event handling in a browser. The Bokeh Python library (and libraries for
-other languages such as R, Scala, and Julia), are primarily a means to interact
-with BokehJS conveniently at a high level, without needing to explicitly worry
-about JavaScript or web development.
+BokehJS is a client-side library that lets you create interactive plots and
+applications. It takes care of drawing, rendering, and event handling. The 
+Bokeh Python library (and libraries for other languages such as R, Scala, 
+and Julia) enables convenient high-level interaction with BokehJS, so you 
+don't have to worry about JavaScript or web development.
 
-However, BokehJS has its own API, and it is possible to do pure JavaScript
-development using BokehJS directly. Additionally, :ref:`userguide_extensions`
-with custom models typically requires interacting with BokehJS directly as well.
+However, BokehJS also has its own API that lets you do pure JavaScript development 
+using BokehJS directly. Additionally, :ref:`userguide_extensions` with custom 
+models typically require direct interaction with BokehJS.
 
 .. warning::
-    The BokehJS APIs is still considered experimental, and may undergo changes
-    in future releases.
+    The BokehJS APIs is still in development and may undergo changes in future
+    releases.
 
 Obtaining BokehJS
 -----------------
 
-BokehJS is made available via CDN and ``npm``. See the :ref:`install_bokehjs`
+BokehJS is available via CDN and ``npm``. See the :ref:`install_bokehjs`
 section of the :ref:`installation` page for more details.
 
 
 .. _userguide_bokehjs_models:
 
-Low-Level Models
+Low-level models
 ----------------
 
-The low-level models for building up plots and applications (e.g. guides
-and glyphs and widgets, etc.) generally match the Bokeh Python models
-exactly. Accordingly, the :ref:`refguide` is a primary resource for
-answering questions about BokehJS models, even though it is presented
-from a Python perspective.
+Generally, the low-level models for plots and applications (e.g. guides,
```

Comment:
To improve legibility, Google's style guide encourages using something like "for example" instead of the Latin "e.g." (https://developers.google.com/style/abbreviations?hl=vi#dont-use)